data-table/table.js

```diff
@@ -149,12 +149,68 @@ function onColumnEntryBoxEnter(event, boxValue) {
      */
 }
 
-function addColumns(numberOfColumns, index) {
-    /**
-     * Adds columns from existing table. Will probably call
-     * createEntryCell(). Maybe should also accept index as an argument?
-     * TODO: Fill this out
-     */
+
+/**
+ * Adds colunms to the table.
```

Comment:
```suggestion
 * Adds columns to the table.
```

(note: here and elsewhere, do a find/replace)